.woodpecker.yml

```diff
@@ -279,8 +279,10 @@ pipeline:
 services:
   database:
     image: postgres:15.2-alpine
+    volumes:
+      # Needed to ensure Lemmy functions without SUPERUSER
+      - ./docker/init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh
```

Comment:
This results in an error "Insufficient privileges to use volumes". See [here](https://woodpecker-ci.org/docs/usage/volumes). So its possible to enable this with extra permissions, but it seems like a security risk in an open source project where anyone can trigger a CI run. Is there another way to do this without volumes?

Reply:
Ouch. Also,

> Please note that Woodpecker mounts volumes on the host machine.

so it's likely that this path will have to be different for each runner host.

The only two workarounds I can think of are:

1. Give up
2. Make a custom Dockerfile with `FROM postgres:15-alpine` and `COPY` in the init script to the right place, then use that image here

Reply:
Tbh I would merge it without the CI change. Thats whats important anyway, and we only added something requiring superuser a single time, so its not exactly a common problem.

Reply:
Yeah, reasonable. Option 2 is pretty messy. I'll just drop the second commit then.